paymentsheet-example/src/androidTestDebug/java/com/stripe/android/TestMultiStepFieldsReloaded.kt

```diff
@@ -0,0 +1,207 @@
+package com.stripe.android
+
+import androidx.compose.ui.test.assertContentDescriptionEquals
+import androidx.compose.ui.test.junit4.createEmptyComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performTextInput
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import com.stripe.android.paymentsheet.model.SupportedPaymentMethod
+import com.stripe.android.test.core.AuthorizeAction
+import com.stripe.android.test.core.Automatic
+import com.stripe.android.test.core.Billing
+import com.stripe.android.test.core.Currency
+import com.stripe.android.test.core.Customer
+import com.stripe.android.test.core.DelayedPMs
+import com.stripe.android.test.core.GooglePayState
+import com.stripe.android.test.core.INDIVIDUAL_TEST_TIMEOUT_SECONDS
+import com.stripe.android.test.core.IntentType
+import com.stripe.android.test.core.MyScreenCaptureProcessor
+import com.stripe.android.test.core.PlaygroundTestDriver
+import com.stripe.android.test.core.Shipping
+import com.stripe.android.test.core.TestParameters
+import com.stripe.android.test.core.TestWatcher
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.Timeout
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+class TestMultiStepFieldsReloaded {
+    @get:Rule
+    var globalTimeout: Timeout = Timeout.seconds(INDIVIDUAL_TEST_TIMEOUT_SECONDS)
+
+    @get:Rule
+    val composeTestRule = createEmptyComposeRule()
+
+    @get:Rule
+    val testWatcher = TestWatcher()
+
+    private lateinit var device: UiDevice
+    private lateinit var testDriver: PlaygroundTestDriver
+
+    private val newUser = TestParameters(
+        SupportedPaymentMethod.Bancontact,
+        Customer.New,
+        GooglePayState.Off,
+        Currency.EUR,
+        IntentType.Pay,
+        Billing.Off,
+        shipping = Shipping.Off,
+        delayed = DelayedPMs.Off,
+        automatic = Automatic.On,
+        saveCheckboxValue = false,
+        saveForFutureUseCheckboxVisible = false,
+        useBrowser = null,
+        authorizationAction = AuthorizeAction.Authorize,
+    )
+
+    @Before
+    fun before() {
+        device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
```

Comment:
Seems like we have a lot of duplicate code across our tests. Not in this PR, but maybe we should build a base test case class in the future? I might look into it as a part of my work (or add to the problem)

Reply:
yeah, fair point, sounds good